pygmt/datasets/earth_relief.py

```diff
@@ -69,6 +70,7 @@ def _is_valid_resolution(resolution):
     Examples
     --------
 
+    >>> _is_valid_resolution("01d")
     >>> _is_valid_resolution("60m")
```

Comment:
```suggestion

```
Remove?

Reply:
I want to keep it as a test for GMT 6.0.0. When we move to GMT 6.1.0, we definitely will remove it anyway.

Reply:
Ok, fair enough.